Book/php7/zend_engine/zend_compiler.rst

```diff
@@ -1,2 +1,6 @@
 Zend Compiler
 =============
+
+An example of how to add a syntactic feature to PHP (for lexing, adding to PHP's context free grammar, and adding a new opcode) is https://nikic.github.io/2012/07/27/How-to-add-new-syntactic-features-to-PHP.html.
```

Comment:
This one is pretty dated (it's for PHP 5). I'd suggest linking to https://phpinternals.net/articles/implementing_a_range_operator_into_php instead.

Reply:
Done.